### ISSUE TITLE
Expand information shown in the "still looking" list

### ISSUE THREAD

**Issue by brandonhaslegs** [radicle-dev/radicle-upstream#1873]
![image](https://user-images.githubusercontent.com/4406983/117640773-aad84900-b185-11eb-8412-eb35cb9e252b.png)

If you're connected to a seed node and are pulling a project from that seed, it should happen pretty much instantly. However, if there are any issues, and the project is not immediately replicated, we should show the user what the state of the replication is.

- States are:
  - Asked
  - Found
  - Cloning
  - Cloned
  - Failed (not sure what makes something fail) 
  - Timed Out (we make multiple attempts to query the network for a project, if after the last attempt, we didn't hear back from any peer, it goes to Timed Out)

**Comment by FintanH** [comment]
> Failed (not sure what makes something fail)

Failed means there was an attempt to clone from a peer and that attempt failed. So theoretically, you could show the `PeerId`s that were attempted and failed on.

> Timed Out (we make multiple attempts to query the network for a project, if after the last attempt, we didn't hear back from any peer, it goes to Timed Out)

It also applies to clone attempts (as well as query attempts). So if my client attempted to clone 4 times and the limit was 3, then it "timed out". 

Also, I think the current limits are set to "infinite" so they'll never time out :)

ALSO, the information of how many query and clone attempts are attached to the data so that might be useful to show in the UI. Something to consider at least :)

**Comment by rudolfs** [comment]
I wonder if there's an overlap with: https://github.com/radicle-dev/radicle-upstream/pull/1758? \cc @alexjg

**Comment by alexjg** [comment]
Yep, that PR has a screen which shows the state of all outstanding requests, including the reason a request might have failed (the reason is just a string representation of internal error states so might not be particularly illuminating but is helpful for debugging).

Most of the link work for the PR is now merged so I'll be removing the WIP state shortly.
